Formula/mongodb-csfle.rb

```diff
@@ -2,11 +2,21 @@ class MongodbCsfle < Formula
   desc "MongoDB csfle shared library for Client Side Encryption"
   homepage "https://www.mongodb.com/"
 
-  # TODO(MONGOSH-1193): Bump to production version after 6.0 is released
-  url "https://downloads.mongodb.com/osx/mongo_csfle_v1-macos-x86_64-enterprise-6.0.0-rc0.tgz"
-  sha256 "42ca2389a318b5308cdbc8e688e7784bc2da3f3ea3c2d8a48be4bf9edabacb0b"
+  sha256_aarch64 = "5e5bc88bcadf256c7dcea08a7aa55a552630de6e6919fcf3215b5e8171231c03"
```

Comment:
Do we need these as separate declarations or could we fold them in below?

Reply:
Seems like they are also good to be folded in, done :+1: